x-pack/plugins/cases/public/components/all_cases/table_filters.test.tsx

```diff
@@ -764,6 +766,7 @@ describe('CasesTableFilters ', () => {
         { key: 'status', isActive: false },
         { key: 'severity', isActive: true },
       ];
+
       localStorage.setItem(
```

Comment:
I am a bit(very) confused but how is the local storage relevant here? 

My understanding is it only affects the values in `useAllCasesState` and that is done **above** table filters. That url/localstorage "state" is then passed to the component as props.

Reply:
The local storage here is used to test the filter configuration (which filters are active and which are not) not the state. The `CasesTableFilters` uses the `useFilterConfig` which in turn uses the local storage to load the active filters.